app/components/web-components/au-button/inner-layout.hbs

```diff
@@ -19,6 +19,13 @@
     <WebComponents::AuIcon @name={{@icon}} @size={{@size}}/>
   {{/if}}
 
+  {{#if (eq @layout "icon-left")}}
+    <WebComponents::AuIcon @name={{@icon}} @size={{@size}}/>
+    <div class="auk-button__label">
+      {{yield}}
+    </div>
+  {{/if}}
```

Comment:
Herhaal je hier nu niet de case die al gecovered is met `{{#unless @layout}}`?

Een duidelijkere opbouw is waarschijnlijk om een get function `get layout()` te definieren die de waarde van `this.args.layout` geeft of `icon-left` indien niet gedefinieerd. Op die manier kan de template duidelijk opgebouwd wordn met if-else statements.